client/src/components/IntroInfo/IntroInfo.css

```diff
@@ -31,7 +31,6 @@ p.MuiTypography-root.MuiTypography-body1 {
   font-size: 1.18rem;
```

Comment:
The typography thing here seems to be causing the console error mentioned previously. That said, that's already in, so doesn't necessarily need to be fixed as part of this imho.